### PR TITLE
DSD-826: font size and text wrap updates to the Radio, Checkbox, and Toggle

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-826](https://jira.nypl.org/browse/DSD-826)

## This PR does the following:

- Updates the Radio's and Checkbox's label font size to "label.default".
- Fixes alignment for long labels in the `Checkbox` and `Radio` components.
- Fixes the line height for long labels in the `Toggle` component.

This took longer than expected because the styles did not update as I expected them to. The theme object didn't work and neither did using the `sx` prop over the `__css` prop. The rules had to be added at the top level under the "sizes" property but I don't know why. And some alignment and the line height had to be added directly as Chakra style props to the Chakra components -- and again, I don't know why.

## How has this been tested?

Storybook.

## Accessibility concerns or updates

- N/A

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr897-ukgoy3x5tafozhzfj7wxpyokeoag0cqn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/622a8121af445bb315e2bb32

File: src/components/Toggle/Toggle.tsx [hunk 1]
```diff
@@ -92,6 +92,7 @@ const Toggle = React.forwardRef<HTMLInputElement, ToggleProps>(
             isRequired={isRequired}
             ref={ref}
             size={size === ToggleSizes.Large ? "lg" : "sm"}
+            lineHeight="1.5"
```
Comment (bigfishdesign13) [inline src/components/Toggle/Toggle.tsx:95]: Why does `Toggle` need this, but `Checkbox` and `Radio` do not?

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr897-ukgoy3x5tafozhzfj7wxpyokeoag0cqn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/622a8121af445bb315e2bb32

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
`Checkbox` and `Radio` work well, but `Toggle` needs a bit more work.  The text in `Toggle` is wrapping nicely now, but the vertical alignment also needs to be addressed.  You should be able to use the same `alignItems` update that was applied to the other two, but the `Large` variant will also need padding added to the top of the label text to get it to align better with the increased height of the `Large` variant.  I think `marginTop: 5px` would do it.  It kills me that it does not align with our spacing values, but oh well.

File: src/components/Toggle/Toggle.tsx (hunk 1 above)
Comment (EdwinGuzman) [reply src/components/Toggle/Toggle.tsx:95]: No idea but this is what fixed the text overlap.

**Comment by EdwinGuzman** [comment]
When the text is very long, I don't see much of a difference with the marginTop for the large `Toggle`. If I add that rule and have only one word for the label, it does not align with the `Toggle`'s control anymore. The following is with `marginTop: "5px"` but short label text:

Screen Shot 2022-03-15 at 2 36 05 PM

I don't think we can add a logic check for when there is very long text and adding a marginTop.

**Comment by EdwinGuzman** [comment]
Added the `alignItems` but I don't see much of a change.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr897-ukgoy3x5tafozhzfj7wxpyokeoag0cqn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/622a8121af445bb315e2bb32

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr897-ukgoy3x5tafozhzfj7wxpyokeoag0cqn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/622a8121af445bb315e2bb32

File: src/components/Toggle/Toggle.tsx (hunk 1 above)
Comment (bigfishdesign13) [reply src/components/Toggle/Toggle.tsx:95]: OK, that’s sufficient for me. :)

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr897-ukgoy3x5tafozhzfj7wxpyokeoag0cqn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/622a8121af445bb315e2bb32

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr897-ukgoy3x5tafozhzfj7wxpyokeoag0cqn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/622a8121af445bb315e2bb32

**Comment by EdwinGuzman** [comment]
@bigfishdesign13 the last update looks good.

**Review by bigfishdesign13 — APPROVED** [review]
Teamwork!

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr897-ukgoy3x5tafozhzfj7wxpyokeoag0cqn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/622a8121af445bb315e2bb32

**Pull request merged**